learnosity_sdk/request/init.py

```diff
@@ -151,6 +144,9 @@ def generate_signature(self):
         return self.hash_list(vals)
 
     def validate(self):
+        if self.request_string is None:
+            del self.request_string
```

Comment:
I'd prefer it if we didn't delete variables from the object, I would think just keeping the `request_string` as `None` should be sufficient?